src/_pytest/main.py

```diff
@@ -18,13 +19,25 @@
 from _pytest.outcomes import exit
 from _pytest.runner import collect_one_node
 
-# exitcodes for the command line
-EXIT_OK = 0
-EXIT_TESTSFAILED = 1
-EXIT_INTERRUPTED = 2
-EXIT_INTERNALERROR = 3
-EXIT_USAGEERROR = 4
-EXIT_NOTESTSCOLLECTED = 5
+
+class ExitCode(enum.IntEnum):
+    """
+    encodes the valid exit codes of pytest
```

Comment:
This paragraph is not looking too good:

![image](https://user-images.githubusercontent.com/1085180/59554683-fde6f880-8f7c-11e9-81d6-1268b5852256.png)

Suggestion:

```
Encodes the valid exit codes by pytest.

Currently users and plugins may supply other exit codes as well.
```